core/src/main/java/com/novoda/noplayer/FramesPerSecondCalculator.java

```diff
@@ -0,0 +1,35 @@
+package com.novoda.noplayer;
+
+import java.util.Deque;
+import java.util.LinkedList;
+import java.util.concurrent.TimeUnit;
+
```

Comment:
should this class be in this PR?

Reply:
Yeah, I would consider to remove it @Mecharyry, just to avoid to ship it in what is effectively a non-related change set.

Reply:
728d499